### PR TITLE
refactor: remove NULL; use nullptr instead

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

File: synfig-studio/src/gui/canvasview.cpp [hunk 1]
```diff
@@ -536,7 +536,7 @@ CanvasView::CanvasView(etl::loose_handle<Instance> instance,etl::handle<CanvasIn
 	jack_enabled_in_preview  (false),
 
 	#ifdef WITH_JACK
-	jack_client              (NULL),
+    jack_client              (nullptr),
```
Comment (ice0) [inline synfig-studio/src/gui/canvasview.cpp:539]: tabs?

Comment (rodolforg) [reply synfig-studio/src/gui/canvasview.cpp]: Fixed.

**Comment by rodolforg** [comment]
Removing NULL from core is HUGE. I already did, actually. Should it be included in this PR or another?

**Comment by ice0** [comment]
> Removing NULL from core is HUGE. I already did, actually. Should it be included in this PR or another?

I suggest to split it.

**Comment by rodolforg** [comment]
> I suggest to split it.

Ok. Then I finished this PR and I'll submit another one.

**Comment by ice0** [comment]
@rodolforg 
Please rebase.

**Comment by rodolforg** [comment]
@ice0 done

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**